polyfill/lib/absolute.mjs

```diff
@@ -62,8 +62,12 @@ export class Absolute {
     add = add.plus(bigInt(days).multiply(24 * 60 * 60 * 1e9));
 
     const ns = bigInt(GetSlot(this, EPOCHNANOSECONDS)).plus(add);
+    ES.RejectAbsolute(ns);
```

Comment:
side note: it’s pretty weird to have “reject absolute” rejecting a different thing than what “is absolute” predicates.

Reply:
FWIW, one interpretation of #238 would be to change "IsAbsolute" to "IsTemporalAbsolute" along with the other types, I think that helps a bit with this.